Makefile

```diff
@@ -21,6 +21,9 @@ endif
 TEST_ARTIFACTS_DIR := integration-test/test-artifacts
 BUNDLE_ARCHIVES_DIR := $(TEST_ARTIFACTS_DIR)/archives
 
+BENCH_ARTIFACTS_DIR := runtime/test-artifacts
+BENCH_BUNDLE_DIR := $(BENCH_ARTIFACTS_DIR)/archives
```

Comment:
I think we should avoid duplicating the rootfs archives.

Shall we move this to the root of the repo under a `rootfs-archives` dir (name up for debate)?

It'd require to update both the integration-test and your benchmark code though.